tests/url-generation.js

```diff
@@ -59,6 +59,18 @@ describe("URL generation", function () {
         expect(url).includes(`ik-s=`);
     });
 
+    it('Signed URL with expireSeconds and é in url', function () {
+        const url = imagekit.url({
+            path: "/test_é_path_alt.jpg",
+            signed: true,
+            expireSeconds: 100
+        });
+
+        expect(url).includes(`https://ik.imagekit.io/test_url_endpoint/test_é_path_alt.jpg`);
+        expect(url).includes(`ik-s=`);
```

Comment:
This test case seems incomplete. We can improve it by generating the signature manually in the test case and asserting it against the one returned.

Reply:
updated the test cases